09-features.md

```diff
@@ -44,6 +44,7 @@ The Context column decodes as follows:
 | 22/23 | `option_anchors_zero_fee_htlc_tx` | Anchor commitment type with zero fee HTLC transactions    | IN       | `option_static_remotekey` | [BOLT #3][bolt03-htlc-tx], [lightning-dev][ml-sighash-single-harmful] |
 | 24/25 | `option_route_blinding`           | Node supports blinded paths                               | IN9      | `var_onion_optin`         | [BOLT #4](bolt04-route-blinding)                                      |
 | 26/27 | `option_shutdown_anysegwit`       | Future segwit versions allowed in `shutdown`              | IN       |                           | [BOLT #2][bolt02-shutdown]                                            |
+| 28/29 | `option_attributable_error`      | Can generate/relay attributable errors in `update_fail_htlc`       | IN9      |                   | [BOLT #4][bolt04-attributable-errors]   |
```

Comment:
This needs to be bumped as it conflicts with dual funding.

Reply:
Updated to 36/37